### PR TITLE
3. /api/categories

### PR DESCRIPTION
**Pull request by fatfroggo**

### CODE REVIEW

File: api/app.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const express = require("express")
+
+const { getCategories } = require("./controllers/games.controllers.js")
+
+const app = express();
+app.use(express.json())
```
Comment (hevdevs) [inline api/app.js]: remove express.json() for now, not using it for this endpoint/ticket

File: api/app.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+app.get("/api/categories", getCategories)
+
+app.use((err, req, res, next) => {
+    console.log(err, "internal server error")
```
Comment (hevdevs) [inline api/app.js:10]: remove console.log before merge

**Review by hevdevs — APPROVED** [review]
Couple of minor changes to make before merge, feel free to do so once done with no other PR necessary!

File: api/app.js (hunk 2 above)
Comment (fatfroggo) [reply api/app.js:10]: the console.log is intentional for error handling, not for debugging so i'd like to leave it in the project generally, it only logs when there is an error - is that okay?

**Pull request merged**